modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-gcp-sts"]
+= Preparing cloud credentials for the {cert-manager-operator} with GCP Workload Identity
+
+You can prepare cloud credentials for the {cert-manager-operator} on GCP workload identity by using the `ccoctl` binary.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. The {cert-manager-operator} creates the `CredentialsRequest` object in the `openshift-cloud-credential-operator` namespace. Get the role by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest -n openshift-cloud-credential-operator -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+output
+----
+
+. Annotate the service account by running the following command:
++
+[source,terminal]
+----
+$ oc annotate sa/cert-manager -n cert-manager to-do-value
```

Comment:
Do not need this annotate in GCP workload identity, the other steps should suffice.